02 - Community Expansions/02.04 Auxiliary Tools by Category/Webapp.md

```diff
@@ -0,0 +1,23 @@
+---
+aliases:
+- 
+tags: 
+- seedling 
+publish: true
+---
+
+
+# Webapp
+
+%% Add a description below this line. It doesn't need to be long: one or two sentences should be a good start. %%
+#placeholder/description 
+
+## Tools in this category
+- [[Obsidian Plugin Stats|Obsidian Plugin Stats]]
+%% Add a bullet here and link to the plugins you'd like to categorize! %%
```

Comment:
This is fine, but I just wanted to record my thoughts on why...

Looking at the other categories in this folder, you will see they mostly don't contain explicit links.

They all work via backlinks - see this, for example:
https://publish.obsidian.md/hub/02+-+Community+Expansions/02.04+Auxiliary+Tools+by+Category/iPadOS+Apps

The advantage of this is that if an auxiliary tool changes what platforms it supports, only one file needs to change - the note for the tool - rather than having to keep 2 lots of data synchronised.

However, the template for this category _does_ contain a "Tools in this Category" heading - and a few of the existing files do contain it too.

So I am happy for this link to remain.

Reply:
why isn't the template populated automatically? Because relying on backlink isn't reliable, as by default a new plugin will list all categories. For example, Docker isn't an iPadOS app, but it appears on the backlink of that category.